slopmop/cli/hooks.py

```
@@ -341,96 +481,171 @@ def _hooks_status(project_root: Path, hooks_dir: Path) -> int:
+def _hooks_uninstall(
+    _project_root: Path, hooks_dir: Path, *, global_install: bool = False
+) -> int:
+    """Remove all sm-managed hooks (per-repo, or machine-wide with global)."""
+    target_dir = _global_hooks_dir() if global_install else hooks_dir
+    if not target_dir.exists():
         print("ℹ️  No hooks directory found")
         return 0
 
     removed: list[str] = []
-    hook_types = ["pre-commit", "pre-push", "commit-msg"]
+    hook_types = ["pre-commit", "pre-push"] + _GLOBAL_PASSTHROUGH_HOOKS
```

Ruff (0.15.17)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)